packs/duo/actions/auth.yaml

```diff
@@ -0,0 +1,55 @@
+---
+description: "Carry out an duo auth."
+enabled: true
+entry_point: 'auth.py'
+name: "auth"
+pack: "duo"
+
+parameters:
+  username:
+    type: "string"
+    description: "The Duo username to auth."
+    default: "{{action_context.api_user}}"
+
+  factor:
+    type: "string"
+    description: "Factor to use for authentication."
+    enum:
```

Comment:
It's useful, until you put the action into a Mistral workflow flow where it tracebacks as `{{action_context.api_user}}` is not defined.